verticalLine.lua

```diff
@@ -0,0 +1,28 @@
+local PrimeUI = require "util" -- DO NOT COPY THIS LINE
+local expect = require "cc.expect".expect -- DO NOT COPY THIS LINE
+-- Start copying below this line. --
+
+--- Draws a vertical line at a position with the specified height.
+---@param win window The window to draw on
+---@param x number The X position of the left side of the line
+---@param y number The Y position of the line
+---@param height number The width/length of the line
+---@param side "right"|"left"|nil THe side of the line, either right or left (defaults to left)
```

Comment:
Could you make this a boolean instead? There's only two possibilities, so using a string is a bit overcomplicated.